features/notifications-apps.yml

```
@@ -0,0 +1,19 @@
+name: Notifications from service workers and installed apps
+# This feature is for the platform-specific nature of notifications support
+# in Chrome Android and Safari iOS, and anchors to the Caniuse feature.
+#
+# See `features/notifications.yml` for the full Notifications feature, which
+# does not align with Caniuse.
+description: The Notifications API, when used from a service worker for Chrome Android or home screen app for Safari iOS, displays messages to the user, typically using the notification system of the host operating system.
```

```suggestion
description: The Notifications API, when used from a service worker (for example on Chrome for Android) or a home screen installed webapp (for example on Safari for iOS), displays messages to the user, typically using the notification system of the host operating system.
```

Sorry for suggesting yet another description change here, and for making this one pretty long, but the current phrasing makes it sound like the feature is **only** supported on Chrome Android and Safari iOS. I added "for example" to make it clearer that the API is actually supported everywhere.

---

Maybe something like this, to show the entry point?

```suggestion
description: Notifications via service worker registration's `showNotification()` method, sent from installed web applications (for example, saved to the Home Screen on Safari for iOS), show a message to the user, typically using the notification system of the host operating system.
```

---

Took @ddbeck's change, as it incorporate's @captainbrosset's callout but w/ fewer parens.